google/cloud/forseti/services/inventory/inventory.py

```diff
@@ -255,7 +255,7 @@ def do_inventory():
                     return result.get_summary()
 
                 except Exception as e:
-                    LOGGER.exception(e)
+                    LOGGER.error(e, exc_info=True)
```

Comment:
According to the answer with the 77 votes, LOGGER.exception(e) is the same as LOGGER.error(e, exc_info=True).  So, please verify why this change would actually help to catch the error.

If that is true, then I would prefer LOGGER.exception because it is more readable, and it allows you to pass in a customer message and still show the stacktrace.

https://stackoverflow.com/questions/5191830/how-do-i-log-a-python-error-with-debug-information

Reply:
Verified exception() is same as error() with exc_info=True, updated it back to exception.

https://github.com/python/cpython/blob/ea68d83933e6de6cabfb115ec1b8888301947369/Lib/logging/__init__.py#L1367